docs/docs/20-quickstart.md

```diff
@@ -489,10 +489,7 @@ the previous section.
     <TabItem value="general" label="Mac, Linux, or WSL" default>
 
     ```shell
-    arch=$(uname -m)
-    [ "$arch" = "x86_64" ] && arch=amd64
-    curl -L -o kargo https://github.com/akuity/kargo/releases/latest/download/kargo-"$(uname -s | tr '[:upper:]' '[:lower:]')-${arch}"
-    chmod +x kargo
+    brew install akuity/tap/kargo
```

Comment:
I've been thinking about this particular change and I think we would be better off keeping the original install instructions until we're supremely confident that the automated formula updates are working as expected. Let's roll this one change back and re-add it at a later date.